libs/localization/assets/locales/en/sheet.json

```diff
@@ -152,7 +152,7 @@
   "bonusScaling": {
     "skill_": "Ele. Skill Bonus Scaling",
     "atkInc": "ATK Inc. Bonus Scaling",
-    "percentStat": "{{percent}} {{stat}}"
```

Comment:
Do we really need a translated string for this? why not add the % ourselves in the sheet?

Reply:
i think you commented on an old version of the file. Anyways, I'm actually doing this to give the option for translations to add connectives (pretty sure some languages will want to do the equivalent of adding an "of" in there.

uhhh i think i have other reasons but i can only think of this one right now and i think it's good enough